portal/templates/redesign/teach_new/class_new.html

```diff
@@ -0,0 +1,143 @@
+{% extends 'redesign/teach_new/base_class_edit.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<section class="background">
+    <h2>Mr Holland, Test class 1 (EU184)</h2>
+    <h4 class="col-sm-6 col-center">Here you can view and manage all of your students within each of your classes. We&rsquo;ve created reminder cards which includes each
+        students login details for you to print out, meaning they won&rsquo;t forget and you don&rsquo;t have to remember this for them. You can add new students, transfer
+        existing students to another one of your classes or to another teacher within your school or club, or remove students altogether.</h4>
+</section>
+
+<div class="background col-sm-8 col-center">
+    <div class="row">
+        <div class="col-sm-6">
+            <h3>Current students</h3><br>
+            <p class="body-text">Select an individual student to change their details, including their name and password.</p><br>
+            <p class="body-text">Select multiple students using the checkboxes to reset their passwords, move them to another class, release them from your school and
+                make them an independent Code for Life user, or delete them permanently.</p>
+        </div>
+        <div class="col-sm-6">
+            <table class="header-secondary data-quaternary">
+                <tr>
+                    <th class="cell-center">
+                        <p class="body-text">Select</p>
+                        <input type="checkbox" name="vehicle" value="">
+                        <p class="small-text">Select all</p>
+                    </th>
+                    <th><p class="body-text">Student details</p></th>
+                    <th>
+                        <p class="body-text">Status</p>
+                        <div class="circle circle--green"></div><p class="small-text">Logged in</p>
+                        <div class="circle circle--red"></div><p class="small-text">Logged out</p>
+                    </th>
+                </tr>
+                <tr>
+                    <td class="cell-center"><input type="checkbox" name="vehicle" value=""></td>
+                    <td>
+                        <p class="small-text">Ben Doorhandle</p>
```

Comment:
Well, obviously.  Thing is, when you do that, it becomes harder to spot what's dummy text from your testing database, and what's dummy text from your design.  That means it's easy to miss which bits of your page aren't properly automated.